### ISSUE TITLE
Не обрабатываются ошибки в процессе разбэкапа базы

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#1783]
Originally reported on Google Code with ID 1783

```
У меня есть база, в процессе обработки которой в лог пишется следующее:

15:03:43: Копирование файла базы данных
15:05:01: Проверка базы данных
15:05:18: Проверка базы данных на NULL значения
15:05:37: Комментирование хранимых процедур
15:05:39: Комментирование триггеров
15:05:45: Архивирование представлений и вычисляемых полей
15:05:46: Архивация базы данных
15:07:08: Восстановление базы данных
gbak:cannot commit index RDB$FOREIGN394 
gbak: ERROR:unsuccessful metadata update 
gbak: ERROR:    partner index segment no 1 has incompatible data type 
gbak:Database is not online due to failure to activate one or more indices. 
gbak:Run gfix -online to bring database online without active indices. 
15:14:43: Восстановление представлений и вычисляемых полей
15:14:44: Удаление комментариев из хранимых процедур
15:14:47: Удаление комментариев из триггеров

Т.е. есть какие-то ошибки, но процесс не прерывается и по его окончании
выдается следующая информация:

Процесс завершен успешно
Оригинальная база данных сохранена под именем:
  D:\convert\GDBASE.bak
Сконвертированная база данных сохранена под именем:
  D:\convert\GDBASE.FDB

```

Reported by `gs1994` on 2010-01-04 12:57:22

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2010-01-17 16:33:29
- **Labels added**: FDBConvert
- **Labels removed**: FBDBConvert

**Comment by gsbelarus** [comment]
Reported by `flakekun` on 2010-01-26 13:33:06
- **Status changed**: `Duplicate`
- **Merged into**: #1814
